isis/src/base/objs/Spice/Spice.cpp

```diff
@@ -374,13 +374,20 @@ namespace Isis {
                                                   ltState, targetRadius);
 
     m_sunPosition = new SpicePosition(10, m_target->naifBodyCode());
-    
+
 
     // Check to see if we have nadir pointing that needs to be computed &
-    // See if we have table blobs to load 
+    // See if we have table blobs to load
     if (m_usingAle) {
-      m_sunPosition->LoadCache(isd["sun_position"]);
-      m_bodyRotation->LoadCache(isd["body_rotation"]);
+      m_sunPosition->LoadCache(isd["SunPosition"]);
+      if (m_sunPosition->cacheSize() > 3) {
+        m_sunPosition->Memcache2HermiteCache(0.01);
+      }
+      m_bodyRotation->LoadCache(isd["BodyRotation"]);
+      m_bodyRotation->MinimizeCache(SpiceRotation::DownsizeStatus::Yes);
```

Comment:
Did you end up verifying that `SpiceRotation::DownsizeStatus::No` isn't used anywhere? Because this would over-ride that, right?

Reply:
I know that Done isn't being used anywhere if that's what you are talking about. I think No is used in a couple of places.